src/garage/torch/algos/mtsac.py

```diff
@@ -93,8 +97,9 @@ def __init__(
             reward_scale=1.0,
             optimizer=torch.optim.Adam,
             steps_per_epoch=1,
-            num_evaluation_trajectories=5,
-    ):
+            # yapf: disable
```

Comment:
why?

Reply:
The CI was complaining that the placement of the `):` characters. It called it invalid syntax whenever YAPF would place it automatically.

Reply:
just place `# yapf: disable` on line 101 to prevent yapf from messing with the entire args block.